### PR TITLE
Don't warn if files with tests for bindings are modified

### LINKED ISSUE #49: Add TestBinding.webidl and testbinding.rs to the test check
Actually if you modify script code you can also add tests to these files:

 * `components/script/dom/testbinding.rs`
 * `components/script/dom/webidls/TestBinding.webidl`

Right now highfive warns you even though you've added tests there (see https://github.com/servo/servo/pull/9304) for example.

Should be easy to check if these files are modified too to prevent the false warning.

### PR DESCRIPTION
**Pull request by flacerdk**
I can't say I really understood how the tests work--hard to tell what's being tested when there's no live repository--, but I went ahead and added one anyway. Fixes #49.

### CODE REVIEW

File: handlers/missing_test/__init__.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 class MissingTestHandler(EventHandler):
     COMPONENT_DIRS_TO_CHECK = ('layout', 'script', 'gfx', 'style', 'net')
     TEST_DIRS_TO_CHECK = ('ref', 'wpt', 'unit')
+    TEST_FILES_TO_CHECK = ('components/script/dom/testbinding.rs',
+                           'components/script/dom/webidls/TestBinding.webidl')
```
Comment (jdm) [inline handlers/missing_test/__init__.py]: Let's also add TestBindingProxy.webidl, TestBindingIterable.webidl, TestBindingPairIterable.webidl, testbindingproxy.rs, testbindingiterable.rs, and testbindingpairiterable.rs.

**Comment by jdm** [comment]
Thanks for doing this! The test is checking the right thing, so thanks for adding it.

**Comment by flacerdk** [comment]
Great! I'll add the other files, then.

**Comment by Wafflespeanut** [comment]
Looking good. Can you please squash the commits?

**Comment by flacerdk** [comment]
Done!

**Comment by jdm** [comment]
Thanks!

**Pull request merged**